cloud-service-providers/azure/promptflow/README.md

```diff
@@ -0,0 +1,189 @@
+# End to End LLM App development with Azure AI Studio, Prompt Flow and NIMs
+
+ ![tl:dr](./images/contoso-chat-nim.png)
+
+**Table Of Contents**
+
+1. [Pre-Requisites](#1-pre-requisites)
+2. [Environment](#2-environment)
+3. [Provision Azure Resources](#3-create-azure-resources)
+    - 3.1 [Authenticate With Azure](#31-authenticate-with-azure)
+    - 3.2 [Run Provisioning Script](#32-run-provisioning-script)
+    - 3.3 [Verify config.json setup](#33-verify-configjson-setup)
+    - 3.4 [Verify .env setup](#34-verify-env-setup)
+    - 3.5 [Verify local Connections](#35-verify-local-connections-for-prompt-flow)
+    - 3.6 [Verify cloud Connections](#36-verify-cloud-connections-for-prompt-flow)
+4. [Populate With Your Data](#4-populate-with-sample-data)
+5. [Build Your Prompt Flow](#5-building-a-prompt-flow)
+    - 5.1 [Explore contoso-chat Prompt Flow](#51-explore-the-contoso-chat-prompt-flow)
+    - 5.2 [Run The Prompt Flow](#52-run-the-prompt-flow)
+
+
+## 1. Pre-Requisites
+
+- **Azure Subscription** - [Signup for a free account.](https://azure.microsoft.com/free/)
+- **Visual Studio Code** - [Download it for free.](https://code.visualstudio.com/download)
+- **GitHub Account** - [Signup for a free account.](https://github.com/signup)
+- **Access to Azure Open AI Services** - [Learn about getting access.](https://learn.microsoft.com/legal/cognitive-services/openai/limited-access)
+- **Ability to provision Azure AI Search (Paid)** - Required for Semantic Ranker
+
+## 2. Environment
+
+1. Clone the repo and open it in VS code
+
+    ```bash
+    git clone ssh://git@gitlab-master.nvidia.com:12051/ai-sae/nim-on-azure.git
```

Comment:
Is this externally available?

Reply:
thanks for pointing it out. Corrected it